lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -96,8 +100,19 @@ def safe_method(method_name, node)
           end
         end
 
+        def check_localtime(node)
+          p = node
```

Comment:
`p` is not a very good name for a variable. Btw, it's not clear at all why you decided to introduce it.